src/Raven.Server/Documents/AI/AiAssistant/Handlers/AiAssistantHandler.cs

```diff
@@ -0,0 +1,37 @@
+﻿using System.Threading.Tasks;
+using Raven.Server.Documents.AI.AiAssistant.Handlers.Processors;
+using Raven.Server.Routing;
+using Raven.Server.Web;
+
+namespace Raven.Server.Documents.AI.AiAssistant.Handlers;
+
+public class AiAssistantHandler : ServerRequestHandler
+{
+    [RavenAction("/assistant/assist", "POST", AuthorizationStatus.ValidUser, EndpointType.Read)]
+    public async Task Assist()
+    {
+        using (var processor = new AiAssistantAssistProcessor(this))
```

Comment:
Wondering if we should log an audit entry, upon docs/data being sent to AI. Thoughts?